<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Recover_Password</class>
 <widget class="QMainWindow" name="Recover_Password">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="LoginMenu">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>681</width>
      <height>421</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="Input_ID_EditText">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>160</y>
       <width>371</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>请输入注册身份证号码</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="Recover_Button">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>340</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>OpenHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #0076b4;
    border: 1px solid #dcdfe6;
    padding: 10px;
    border-radius: 5px;
	color: rgb(255, 255, 255);
	font: 600 10pt &quot;Segoe UI Variable Text Semibold&quot;;
}

QPushButton:hover {
    background-color: #ecf5ff;
    color: #409eff;
}
</string>
     </property>
     <property name="text">
      <string>重置</string>
     </property>
    </widget>
    <widget class="QLabel" name="Welcome_Label">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>351</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; font-weight:700; color:#0974ae;&quot;&gt;重置密码&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="Wrong_PhoneNum_Label">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>90</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:700; color:#ff0000;&quot;&gt;× 请输入正确的手机号！ &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="Login_Menu_Image_Label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>201</width>
       <height>381</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Close_Button">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>0</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border-style:outset;
background-color: rgba(255, 255, 255, 0);
border-radius:0px
}

QPushButton:hover{ 
	background-color: rgb(255, 0, 0);
}
 
QPushButton:pressed{
	background-color: rgb(220,92,102);
}</string>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Hide_Button">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>0</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border-style:outset;
background-color: rgba(255, 255, 255, 0);
border-radius:0px
}

QPushButton:hover{ 
	background-color: rgb(239, 231, 231);
}
 
QPushButton:pressed{
	background-color: rgb(243,234,234);
}</string>
     </property>
     <property name="text">
      <string>—</string>
     </property>
    </widget>
    <widget class="QLabel" name="Not_Input_PhoneNum_Label">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>90</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:700; color:#ff0000;&quot;&gt;× 请输入手机号！ &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="Not_Input_Password_Label">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>210</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:700; color:#ff0000;&quot;&gt;  × 请输入密码！ &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Input_Password_EditText">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>220</y>
       <width>371</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::PasswordEchoOnEdit</enum>
     </property>
     <property name="placeholderText">
      <string>请设置3-16位密码，至少包含数字和字母</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="Input_PhoneNum_EditText">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>100</y>
       <width>371</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>请输入注册手机号</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="Input_Password_Again_EditText">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>280</y>
       <width>371</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::PasswordEchoOnEdit</enum>
     </property>
     <property name="placeholderText">
      <string>请再次输入密码</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="Not_Input_ID_Label">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>150</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:700; color:#ff0000;&quot;&gt;× 请输入身份证号！ &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="Not_Input_Password_Label_2">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>270</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:700; color:#ff0000;&quot;&gt;× 请再次输入密码！ &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="Return_Login_Label">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>320</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>OpenHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{background-color:transparent;}
QLabel:hover {color:rgb(51, 144, 255);}
</string>
     </property>
     <property name="text">
      <string>想起了密码？返回登录</string>
     </property>
    </widget>
    <widget class="QLabel" name="Wrong_ID_Label">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>150</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:700; color:#ff0000;&quot;&gt;× 请输入正确的身份证号！ &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="Wrong_Password_Length_Label">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>210</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:700; color:#ff0000;&quot;&gt;× 密码长度必须为3-16位！ &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="Wrong_Password_Type_Label">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>210</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:700; color:#ff0000;&quot;&gt;× 密码至少要包含数字和字母！ &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="Password_Different_Label">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>270</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:700; color:#ff0000;&quot;&gt;× 两次输入的密码不一致！ &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="Not_Exist_PhoneNum_Label">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>90</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:700; color:#ff0000;&quot;&gt;× 手机号不存在！ &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="Not_Exist_ID_Label">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>150</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:700; color:#ff0000;&quot;&gt;× 身份证号不存在！ &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="Different_PhoneNum_Label">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>90</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:700; color:#ff0000;&quot;&gt;× 注册证件信息不一致！&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="Different_ID_Label">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>150</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:700; color:#ff0000;&quot;&gt;× 注册证件信息不一致！&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <zorder>Input_Password_EditText</zorder>
    <zorder>Input_ID_EditText</zorder>
    <zorder>Recover_Button</zorder>
    <zorder>Welcome_Label</zorder>
    <zorder>Login_Menu_Image_Label</zorder>
    <zorder>Close_Button</zorder>
    <zorder>Hide_Button</zorder>
    <zorder>Input_PhoneNum_EditText</zorder>
    <zorder>Input_Password_Again_EditText</zorder>
    <zorder>Return_Login_Label</zorder>
    <zorder>Wrong_PhoneNum_Label</zorder>
    <zorder>Wrong_ID_Label</zorder>
    <zorder>Wrong_Password_Type_Label</zorder>
    <zorder>Wrong_Password_Length_Label</zorder>
    <zorder>Password_Different_Label</zorder>
    <zorder>Not_Input_ID_Label</zorder>
    <zorder>Not_Input_Password_Label</zorder>
    <zorder>Not_Input_Password_Label_2</zorder>
    <zorder>Not_Input_PhoneNum_Label</zorder>
    <zorder>Not_Exist_PhoneNum_Label</zorder>
    <zorder>Not_Exist_ID_Label</zorder>
    <zorder>Different_PhoneNum_Label</zorder>
    <zorder>Different_ID_Label</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>684</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>Input_PhoneNum_EditText</tabstop>
  <tabstop>Input_ID_EditText</tabstop>
  <tabstop>Input_Password_EditText</tabstop>
  <tabstop>Input_Password_Again_EditText</tabstop>
  <tabstop>Recover_Button</tabstop>
  <tabstop>Hide_Button</tabstop>
  <tabstop>Close_Button</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>Input_PhoneNum_EditText</sender>
   <signal>textEdited(QString)</signal>
   <receiver>Recover_Password</receiver>
   <slot>on_Input_PhoneNum_EditText_textEdited()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>355</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>739</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Input_ID_EditText</sender>
   <signal>textEdited(QString)</signal>
   <receiver>Recover_Password</receiver>
   <slot>on_Input_ID_EditText_textEdited()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>435</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>759</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Input_Password_EditText</sender>
   <signal>textEdited(QString)</signal>
   <receiver>Recover_Password</receiver>
   <slot>on_Input_Password_EditText_textEdited()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>483</x>
     <y>261</y>
    </hint>
    <hint type="destinationlabel">
     <x>805</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Input_Password_Again_EditText</sender>
   <signal>textEdited(QString)</signal>
   <receiver>Recover_Password</receiver>
   <slot>on_Input_Password_Again_EditText_textEdited()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>463</x>
     <y>326</y>
    </hint>
    <hint type="destinationlabel">
     <x>805</x>
     <y>357</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Recover_Button</sender>
   <signal>clicked()</signal>
   <receiver>Recover_Password</receiver>
   <slot>on_Recover_PushButton_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>420</x>
     <y>384</y>
    </hint>
    <hint type="destinationlabel">
     <x>761</x>
     <y>424</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_Hide_Button_clicked()</slot>
  <slot>on_Close_Button_clicked()</slot>
  <slot>on_Input_PhoneNum_EditText_textEdited()</slot>
  <slot>on_Input_ID_EditText_textEdited()</slot>
  <slot>on_Input_Password_EditText_textEdited()</slot>
  <slot>on_Input_Password_Again_EditText_textEdited()</slot>
  <slot>on_Recover_PushButton_clicked()</slot>
 </slots>
</ui>
